src/EnergyPlus/VentilatedSlab.cc

```diff
@@ -4515,12 +4340,12 @@ namespace VentilatedSlab {
         } else {
             if ((ventSlab.SysConfg == VentilatedSlabConfig::SlabOnly) || (ventSlab.SysConfg == VentilatedSlabConfig::SeriesSlabs)) {
                 state.dataLoopNodes->Node(FanOutNode) = state.dataLoopNodes->Node(AirOutletNode);
-                state.dataHeatBalFanSys->QRadSysSource(SurfNum) = 0.0;
+                state.dataHeatBalFanSys->QRadSysSource = 0.0;
```

Comment:
Reducing SurfNum showed this dangling use of that local (and below at 4348). I suspect the value of SurfNum here would either be 0 or the last surface in the surface list from a previous loop. It seemed wrong so I changed it to be what I thought it should be (i.e., all surface radiation = 0). @RKStrand ?

Reply:
Thanks for sending this my way to take a look at.  I see now why this won't compile--not sure how it's been compiling at all before you made this change.  This is definitely a bug that needs to be fixed.

The problem is that it is not as simple as getting rid of the (SurfNum).  The reason that should not be done is because that array includes the source/sink inside that surface and could be affected by other systems (like low temp radiant).  Zeroing them all out would basically shut down every system.  What should have happened here is that the QRadSysSource array should be zeroed out only for the surfaces that are associated with this ventilated slab (which, given were these errors are happening in the code, is being turned off for no flow).

So, I think what each of these two problem lines of code need to be replaced with is:

        for (RadSurfNum = 1; RadSurfNum <= ventSlab.NumOfSurfaces; ++RadSurfNum) {
            SurfNum = ventSlab.SurfacePtr(RadSurfNum);
            state.dataHeatBalFanSys->QRadSysSource(SurfNum) = 0.0;
        }

That would then zero out the source/sink term for each of the surfaces associated with the ventilated slab without messing with any other surfaces that might be served by a different ventilated slab, radiant system, etc. that could also be setting values in the QRadSysSource array.  Does this make sense/help?